src/components/Admin/AllDevices.vue

```diff
@@ -45,91 +45,262 @@
     </v-card>
     <v-card>
       <v-card-title>
-        <v-text-field
-          v-model="search"
-          append-icon="mdi-magnify"
-          label="Search"
-          single-line
-          hide-details
-        />
+        <v-spacer />
       </v-card-title>
       <v-data-table
-        :search="search"
         :headers="headers"
         :items="devices"
+        :search="search"
+        sort-by="calories"
```

Comment:
Calories?
Missing ':' so it's not even looking for a prop
```suggestion
        :sort-by=""
```

Reply:
My bad - I forgot to delete part of the example I modified. I'll try to be more careful next time.